src/loc/LocDetailsTab.tsx

```diff
@@ -235,13 +236,13 @@ export function LocDetailsTabContent(props: ContentProps) {
         await mutateLocState(async current => {
             if(current instanceof RejectedRequest) {
                 const next = await current.rework();
-                navigate(backPath);
+                navigate(locDetailsPath(current.locId, current.data().locType));
```

Comment:
Is the navigate still needed? Shouldn't the screen refresh automatically given the state is refreshed?

Reply:
Yes you're right, I remove it.